.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Please note that pull requests without a rationale and clear improvement are
+closed immediately.
+
+Bitcoin Core has a thorough review process and even the most trivial change
+needs to pass a lot of eyes and requires non-zero or even substantial time
+effort to review. There is a huge lack of active reviewers on the project, so
+patches often sit for a long time.
+
+Please provide clear motivation for your patch and explain how it improves
+Bitcoin Core user experience or Bitcoin Core developer experience
+significantly.
```

Comment:
Not sure if I'm reading this right, but is the intention to say that a change needs to

- significantly improve user experience
or
- significantly improve developer experience

or is it more that a change has to 

- improve user experience
or
- significantly improve developer experience

Because the way I read it is that _significantly_ applies to both terms, but I imagine that _significantly_ is only supposed to apply to the second term. Because why would you turn down a papercut fix that smoothes the UX a little?

IOW, maybe this could be rephrased to be less ambiguous